caproto/_broadcaster.py

```diff
@@ -52,6 +53,7 @@ def __init__(self, our_role, protocol_version=DEFAULT_PROTOCOL_VERSION):
             dont_clash_with=self.unanswered_searches,
         )
         self.log = logging.getLogger(f"caproto.bcast")
+        self.beacon_log = logging.getLogger(f'caproto.bcast.beacon')
```

Comment:
The f-string does not bring any benefit here, I'd just remove it.
```suggestion
        self.beacon_log = logging.getLogger('caproto.bcast.beacon')
```